frontend/src/Interfaces/Schedule/Schema/schedule.ts

```
@@ -0,0 +1,50 @@
+import { Schedule } from '../Types/Schedule'
+
+export type CreateScheduleRequest = Omit<Schedule, 'id'> //201
+
+// Create returns just status 201, no body
+export type CreateScheduleResponse = void
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)